python/mxnet/gluon/data/dataloader.py

```diff
@@ -398,9 +398,7 @@ def _worker_fn(samples, batchify_fn, dataset=None):
     # preserving dataset as global variable can save tons of overhead and is safe in new process
     global _worker_dataset
     batch = batchify_fn([_worker_dataset[i] for i in samples])
-    buf = io.BytesIO()
-    ForkingPickler(buf, pickle.HIGHEST_PROTOCOL).dump(batch)
-    return buf.getvalue()
+    return batch
```

Comment:
pickle's protocol isn't quite consistent across versions.
this may or may not be a problem, however I don't have any benchmark result to support either one.

Reply:
Yes. I am working on it. I found the performance of PyTorch is faster than that of MXNet, and I am looking for where the difference is.